docs/make.jl

```diff
@@ -73,8 +73,7 @@ makedocs(
             "reference/advanced.md",
         ],
         "refindex.md",
-    ],
-    strict = true,
```

Comment:
I'm not happy at all about this: #569.

Reply:
Should we add an `API` page in the docs listing all the low level methods ?
How can I detect them automatically ?
It is not conceivable to do it by hand.

Reply:
Ask Documenter to collect the functions in a given module: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/7721c30465a397cba49ee75a7647c4fbac403d96/docs/src/index.md#functions

Reply:
Fixed, by adding `api.md`.